backend/api/appsettings.json

```diff
@@ -58,7 +58,7 @@
       "Logout": "/protocol/openid-connect/logout",
       "Register": "/protocol/openid-connect/registrations",
       "Token": "/protocol/openid-connect/token",
-      "TokenIntrospect": "/protocol/openid-connect/token/introspect",
+      "TokenIntrospect": "/protocol/openid-connect/token/introspect/fake",
```

Comment:
double checking that this URL is correct?

Reply:
this is unused, but will change it back to match the endpoint.